src/debug.nit

```diff
@@ -3,6 +3,10 @@ import api
 redef class AppConfig
 	# The default player, if any, that will be logged on a new session
 	var debug_player: String is lazy do return value_or_default("app.debug.player", "")
+
+	init do
+		engine_map["nitc"] = new NitcEngine
```

Comment:
While we're on the subject of a dropdown for languages, since the information of both the engine and the language are available, we can consider printing something along the lines of:

* nitc for Nit
* pep8term for Pep/8